e2e-tests/tfm/cypress/e2e/journeys/deal-cancellation/check-details.spec.js

```diff
@@ -46,15 +44,15 @@ context('Deal cancellation - check details', () => {
       cy.clickContinueButton();
 
       cy.url().should('eq', relative(`/case/${dealId}/cancellation/bank-request-date`));
-      cy.keyboardInput(bankRequestDatePage.bankRequestDateDay(), dateConstants.todayDay);
-      cy.keyboardInput(bankRequestDatePage.bankRequestDateMonth(), dateConstants.todayMonth);
-      cy.keyboardInput(bankRequestDatePage.bankRequestDateYear(), dateConstants.todayYear);
+
+      cy.keyboardInput({ completeDateFormFields: 'bank-request-date' });
```

Comment:
This should be: 

`cy.completeDateFormFields({ idPrefix: 'bank-request-date' });` instead

Reply:
(think this one still needs changing - still says cy.keyboardInput)